src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpContextOfT.cs

```diff
@@ -19,10 +19,10 @@ public IISHttpContextOfT(PipeFactory pipeFactory, IHttpApplication<TContext> app
             _application = application;
         }
 
-        public override async Task ProcessRequestAsync()
```

Comment:
hmm I don’t think this makes sense. Should we just flow the exception? Task\<T\> is very hard to cache

Reply:
Do you want me to rethrow the exception if there was one, and then catch in the HandleRequest method callback?